packages/shared/components/Proposals.svelte

```diff
@@ -5,12 +5,12 @@
     import { IProposal } from '@contexts/governance/interfaces'
     import { proposalFilter } from '@contexts/governance'
     import { isVisibleProposal } from '@contexts/governance/utils/isVisibleProposal'
+    import { sortProposals } from '@contexts/governance/utils/sortProposals'
```

Comment:
Both modules can be added to the utils index and then imported in the same line
```suggestion
    import { isVisibleProposal, sortProposals } from '@contexts/governance/utils'
```